introc/language_facilities.tex

```diff
@@ -532,7 +532,7 @@ \subsection{C data types}
 \begin{enumerate}
 \item \keyword{char} Represents exactly one byte of data.
   The number of bits in a byte might vary.
-  \keyword{unsigned char} and \keyword{signed char} means the exact same thing.
+  \keyword{unsigned char} and \keyword{signed char} are always the same size, which is true for the \keyword{unsigned} and \keyword{signed} versions of all data types.
```

Comment:
This seems like a good place to insert a link to what the different between signed/unsigned is or maybe an explanation of two's complement.